packages/contentstack-config/src/commands/config/remove/rate-limit.ts

```diff
@@ -0,0 +1,41 @@
+import { cliux, configHandler, FlagInput, flags, isAuthenticated } from '@contentstack/cli-utilities';
+import { Command } from '@contentstack/cli-command';
+import { askOrgID, handleErrorMsg } from '../../../utils/interactive';
+
+export default class RateLimitRemoveCommand extends Command {
+
```

Comment:
@harshithad0703 , description is missing

Reply:
added description